oktapam/constants/descriptions/attributes.go

```diff
@@ -149,7 +149,7 @@ var (
 	RefuseConnections                         = "If `true`, the Gateway refuses connections."
 	RemovedAt                                 = "The UTC time when the resource was removed from parent resource. Format is '2022-01-01 00:00:00 +0000 UTC'."
 	RequirePreauthForCreds                    = "If `true`, requires preauthorization before a User can retrieve credentials to sign in."
-	ResourceGroupID                           = "The UUID of a OktaPA Resource Group."
```

Comment:
So technically the product is "Okta Privileged Access", or OPA. We're trying really hard to avoid calling it Okta PAM.